OKlib.js

```diff
@@ -824,416 +767,378 @@ registerPlugin({
+     * checks if a is greater or equal b
+     * @param {Object} a
+     * @param {Object} b
+     * @return {Boolean} returns true when a is greater or equal b
      */
     function greaterOrEqual(a, b) {
         return (a >= b);
     }
 
     /**
-     * Checks if a is less or equal b
-     * @param  {Object} a
-     * @param  {Object} b
-     * @return {Boolean}   Returns true when a is less or equal b
+     * checks if a is less or equal b
+     * @param {Object} a
+     * @param {Object} b
+     * @return {Boolean} returns true when a is less or equal b
      */
     function lessOrEqual(a, b) {
         return (a <= b);
     }
 
     /**
-     * Checks if one String is contained in another
-     * @param  {String} a
-     * @param  {String} b
-     * @return {Boolean}   Returns true when String b is contained in String a
+     * checks if one string is contained in another
+     * @param {String} a
+     * @param {String} b
+     * @return {Boolean} returns true when string b is contained in string a
      */
     function contains(a, b) {
-        if (a.indexOf(b) !== -1)
-            return true;
+        if (a.includes(b)) return true;
         return false;
     }
 
     /**
-     * Checks if one String is contained in another, ignoring the Case
-     * @param  {String} a
-     * @param  {String} b
-     * @return {Boolean}   Returns true when String b is contained in String a
+     * checks if one string is contained in another, ignoring the case
+     * @param {String} a
+     * @param {String} b
+     * @return {Boolean} returns true when string b is contained in string a
      */
     function containsIgnoreCase(a, b) {
-        if (a.toLowerCase().indexOf(b.toLowerCase()) !== -1)
-            return true;
+        if (a.toLowerCase().includes(b.toLowerCase())) return true;
         return false;
     }
 
     /**
-     * Checks if the value is a Number
-     * @param  {Object}  number Object that should be checked
-     * @return {Boolean}        Returns true if the Object is a Number
+     * checks if the value is a number
+     * @param {Object} number object that should be checked
+     * @return {Boolean} returns true if the object is a number
      */
     function isNumber(number) {
         return !isNaN(number);
     }
 
     /**
-     * Simple Password Generator
-     * @param  {number}  length Password Length
-     * @param  {String}  charset Charset to use for the password generation
-     * @return {String}    Returns a Password with the specified Length
+     * simple password generator
+     * @param {number} length password length
+     * @return {String} returns a password with the specified length
      */
-    function generatePassword(length, charset) {
-        let result = "";
-        if (charset === undefined)
-            charset = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
-        for (let i = 0; i < length; i++)
+    function generatePassword(length) {
```

Comment:
Why remove charset value (+ doc for that parameter)?

Explanation or reverting.

Reply:
I updated from the old oklib version not from your already made changes for v8. The old version didn't have the optional charset included so I didn't have it either. I added it back.